lib/giraph/schema.rb

```diff
@@ -0,0 +1,46 @@
+module Giraph
+  # GraphQL::Schema wrapper allowing decleration of
+  # resolver objects per op type (query or mutation)
+  class Schema < GraphQL::Schema
```

Comment:
Consider making this a delegate class to avoid having to subclass `GraphQL::Schema`
```ruby
class Schema < DelegateClass(GraphQL::Schema)
  def initialize(**args)
    @query_resolver = args.delete(:query_resolver)
    @mutation_resolver = args.delete(:mutation_resolver)
    super(GraphQL::Schema.new(**args))
  end
end
```